index.bs

```diff
@@ -760,7 +760,7 @@ An attribution source is a [=struct=] with the following items:
 : <dfn>event ID</dfn>
 :: A non-negative 64-bit integer.
 : <dfn>attribution destinations</dfn>
-:: An [=ordered set=] of [=sites=].
+:: A [=set=] of [=sites=].
```

Comment:
Does this also affect their appearance in the report? I think keeping it ordered in the report makes sens.

Reply:
Again, this change doesn't *affect* anything, but whether it's relevant is a valid question. I don't think we should ascribe any semantics to the ordering of destinations within a report unless or until such semantics are needed.

Reply:
Re my findings in https://github.com/WICG/attribution-reporting-api/pull/1217#discussion_r1589148599 the use of `ordered set` here actually implies that the report's `attribution_destination` field must have the same ordering as the attributed source's `destination` field, which we do not want, as it reveals (minor) extra information from the source. IMO the field should be sorted by site to avoid leaking anything additional here.

Will update to this effect.

Reply:
Done.